Roles/(Ghosts)/Crewmate/Telepathy.cs

```diff
@@ -0,0 +1,209 @@
+﻿using AmongUs.GameOptions;
+using TOHE.Roles.Core;
+using System;
+using UnityEngine;
+using static TOHE.Options;
+using static TOHE.Translator;
+using static TOHE.MeetingHudStartPatch;
+using static TOHE.Utils;
+
+namespace TOHE.Roles._Ghosts_.Crewmate;
+
+internal class Telepathy : RoleBase
+{
+    //===========================SETUP================================\\
+    private const int Id = 29300;
+    public static bool HasEnabled => CustomRoleManager.HasEnabled(CustomRoles.NotAssigned);
```

Comment:
Erm, What the Sigma? (`CustomRoles.NotAssigned`)

Reply:
oh excuse my lvl 1 gyatt, fixed now